src/parser/Parser.ts

```diff
@@ -2318,28 +2326,39 @@ export class Parser {
         return this.previous();
     }
 
-    private checkPrevious(...tokenKinds: TokenKind[]) {
-        if (this.isAtEnd()) {
+    private checkPrevious(tokenKind: TokenKind) {
+        return this.previous().kind === tokenKind;
+    }
+
+    private check1(tokenKind: TokenKind) {
```

Comment:
Rather than calling this `check1`, how about we rename the original `check` to `checkAny`, and call this one `check`? This would apply to all of the `1` methods you created.